### ISSUE TITLE
Dropbox test : Error reading JObject from JsonReader. Path '', line 0, position 0.

### ISSUE THREAD

**Issue by Najihel** [duplicati/duplicati#3374]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.3.11_canary_2018-09-05
- **Operating system**: Linux Mint 19
- **Backend**: Dropbox

## Description

```
Error while running Ecritel-Laptop
Error reading JObject from JsonReader. Path '', line 0, position 0.
```

## Steps to reproduce

1. Create a job
2. Select Dropbox destination
3. Generate AuthID
4. Press "Test connection"

## Screenshots

![selection_002](https://user-images.githubusercontent.com/1873677/45350405-05274300-b5b4-11e8-9e29-2a540a2aa197.png)
![selection_003](https://user-images.githubusercontent.com/1873677/45350517-3acc2c00-b5b4-11e8-9c15-7ebd7db41666.png)

**Comment by ts678** [comment]
Another report [here](https://forum.duplicati.com/t/one-missing-file-dropbox-destination/4674) with some good historical analysis and logs available for possible clues.

**Comment by warwickmm** [comment]
I believe this was fixed in pull request #4195.
